### PR TITLE
Add missing Distributions methods for KeyedMixtureModel

### PR DESCRIPTION
**Pull request by aisopous**
`Distributions.ncomponents` and `Distributions.probs` are missing.  This MR adds them.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "KeyedDistributions"
 uuid = "2576fb08-064d-4cab-b15d-8dda7fcb9a6d"
 authors = ["Invenia Technical Computing Corporation"]
-version = "0.1.16"
+version = "0.1.17"
```
Comment (rofinn) [inline Project.toml:4]: Yeah, since hasn't hit 1.0 yet I think a patch release makes sense.

File: src/KeyedDistributions.jl [hunk 2]
```diff
@@ -214,6 +214,12 @@ Distributions.sampler(d::KeyedDistribution) = sampler(distribution(d))
 
 Distributions.params(d::KeyedDistOrSampleable) = params(distribution(d))
 
+Distributions.probs(d::KeyedMixtureModel) = probs(distribution(d))
```
Comment (rofinn) [inline src/KeyedDistributions.jl:217]: If we need to add anymore of these we might want to just do it in an eval loop?

**Review by rofinn — APPROVED** [review]
LGTM

**Review by oxinabox — APPROVED** [review]

**Comment by aisopous** [comment]
I don't have merge rights, could someone give me a hand?

**Pull request merged**